README.md

```diff
@@ -123,69 +49,288 @@ GRANT ALL PRIVILEGES ON DATABASE sample TO manager;
 | SENDGRID_FROM_EMAIL               | <sendgridfromemail>  | The e-mail address used for notifications   |
 | LOCAL                             | False                | Tells the site to use external API or mocks |
 
+# Getting Started
+
+## First time here?
+
+The docker image for this branch is created differently from the other branches (`python manage.py runserver 0.0.0.0` is now run inside the image, not as a command on the image in the `docker-compose.yml`). This means that the old containers and images for other branches need to be deleted so the new one can be built.
+
+```
+$ docker ps --all
+CONTAINER ID     IMAGE               COMMAND                  CREATED        STATUS                  PORTS       NAMES
+<container_id>   wagtail-cms_web     "python manage.py ru…"   3 days ago     Exited (0) 3 days ago               wagtail-cms_web_1
+............     .....               "...................."   ..........     .............           27017/tcp   .................
+$ docker rmi -f <container_id>
+$ docker images
+REPOSITORY                            TAG          IMAGE ID       CREATED         SIZE
+wagtail-cms_web                       latest       <image_id>     11 days ago     1.07GB
+...............                       latest       ............   3 weeks ago     ......
+$ docker rm <image_id>
+...
+```
+
+This now means when you run `docker-compose up` docker will rebuild the image using the new commands in the `Dockerfile`, and then build a new container from that
+
+> *REMEMBER:* To run the commands above when switching back to the other branches on the project
+
+## Tests
+
+```
+$ docker container exec -it wagtail-cms_web_1 python manage.py test
```

Comment:
That is the expected action (https://github.com/office-for-students/wagtail-CMS/blob/850b4fa5f5d425a80535e0a56befa50e94ff7d3f/coursefinder/models.py#L262 and https://github.com/office-for-students/wagtail-CMS/blob/850b4fa5f5d425a80535e0a56befa50e94ff7d3f/coursefinder/models.py#L291). Unfortunately it is not documented that the exceptions that they are firing still get logged (that's why they look like they have failed in the terminal).

This whole thing needs to be re-coded so that exceptions bubble up through the code and display the corresponding error page.

Reply:
I have added an additional line to the `README.md` explaining this.